services/cinemahall.py

```diff
@@ -0,0 +1,15 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: int,
```

Comment:
The parameter `hall_name` should be of type `str`, not `int`, as it represents the name of the cinema hall.